src/components/Tutorial/NavigationBar.vue

```diff
@@ -218,6 +218,13 @@ export default {
     .mobile-dropdown-container {
       display: block;
     }
+    /deep/ .nav-title {
+      grid-area: title;
+    }
+
+    /deep/ .pre-title {
```

Comment:
We don't have any place where we set `pre-title` to a `grid-area` but we use it in `grid-template-areas`, which shouldn't work.

Reply:
I mean, this PR works but I don't understand why we use `pre-title` if it's not set to a grid-area.

Reply:
I think its assumed to be in that, because its the first item? The grid follows the natural order of items, top left to bottom right.

I think the bug was happening, because the `.title` was being mushed into the `pre-title` grid-area, because of a `grid-column: 1` rule we had defined earlier. Even though it has a `grid-area: title` that was getting overridden.